pkg/auth/providers/ldap/ldap_client.go

```diff
@@ -492,11 +504,9 @@ func (p *ldapProvider) RefetchGroupPrincipals(principalID string, secret string)
 		distinguishedName,
 		ldapv3.ScopeBaseObject,
```

Comment:
Do you think we should update `ldap.NewSearchRequest` to let people specify scope? That's the only difference I can see between this and the other changed `ldapv3.NewSearchRequest`